slider.js

```diff
@@ -0,0 +1,126 @@
+let thumbsize = 14;
+
+function draw(slider, splitvalue) {
+  /* set function vars */
```

Comment:
I think anybody that heard of programming knows what this section does. Please remove all your unnecessary comments as @AdrianFlorian said.